docs/pages/architecture/tls-routing.mdx

```diff
@@ -112,8 +102,9 @@ Kubernetes client `kubectl` uses HTTPS API and TLS handshake to talk to the API
 server.
 
 As such, it is not possible to request a custom ALPN protocol using `kubectl`.
-Instead, Teleport leverages SNI and sets a `ServerName` prefixed with `kube.`
-when generating a kubeconfig file during `tsh kube login`:
+Instead, Teleport leverages SNI and sets a `ServerName` prefixed with
+`kube-teleport-proxy-alpn.` when generating a kubeconfig file during `tsh kube
+login`:
```

Comment:
I think that `kube-teleport-proxy-alpn` applies only to the Teleport Proxy behind L7 ALB and in case of normal TLS Routing flow we are still using kube. prefix

Reply:
https://github.com/gravitational/teleport/blob/6875666526fe275f65dcc898a91f959e1ccc2139/lib/client/api.go#L880-L905

I think `kube-teleport-proxy-alpn.` is the default regardless of this ALB feature. `kube.` suppose to be removed in 13 or 14?
https://github.com/gravitational/teleport/blob/6875666526fe275f65dcc898a91f959e1ccc2139/api/constants/constants.go#L316-L322

Reply:
Then it makes sense 

> I think kube-teleport-proxy-alpn. is the default regardless of this ALB feature. kube. suppose to be removed in 13 or 14?

yes. The support for the `kube` prefix  should be removed in Teleport v14